src/locales/ko/bgm-name.json

```
@@ -83,9 +83,11 @@
   "battle_aether_grunt": "SM 에테르재단 배틀",
   "battle_skull_grunt": "SM 스컬단 배틀",
   "battle_macro_grunt": "SWSH 트레이너 배틀",
+  "battle_star_grunt": "SV Team Star Battle",
   "battle_galactic_admin": "BDSP 갤럭시단 간부 배틀",
   "battle_skull_admin": "SM 스컬단 간부 배틀",
   "battle_oleana": "SWSH 올리브 배틀",
+  "battle_star_admin": "SV Team Star Boss",
```

```suggestion
  "battle_star_grunt": "SV 스타단 배틀",
  "battle_galactic_admin": "BDSP 갤럭시단 간부 배틀",
  "battle_skull_admin": "SM 스컬단 간부 배틀",
  "battle_oleana": "SWSH 올리브 배틀",
  "battle_star_admin": "SV 스타단 보스 배틀",
```

FYI to KO team, I chekced 'Team Star Boss' using '보스' instead of '간부'